docs/attributes-registry/test.md

```diff
@@ -0,0 +1,37 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Test
+
+## Test Suite Attributes
+
+This group describes attributes specific to [software tests](https://en.wikipedia.org/wiki/Software_testing).
+
+| Attribute                       | Type   | Description                                                                                          | Examples                                                               | Stability                                                        |
+| ------------------------------- | ------ | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `test.suite.case.name`          | string | The fully qualified human readable name of the [test case](https://en.wikipedia.org/wiki/Test_case). | `TestCase1`                                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Should the examples show some fully qualified examples, eg Java class with method `org.example.TestCase1.test1`, JS test `example/tests/Test Case1.test1`?

Reply:
agreed @christophe-kamphaus-jemmic  - I've added some additional examples.